src/3rdparty/kissfft/.travis.yml

```
@@ -14,6 +14,14 @@ addons:
 
 install: true
 
-script:
-  - make all
-  - make testall
+jobs:
+  include:
+    - name: "build (make)"
+      script:
+      - make all
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 8 but found 6

(indentation)